openage/convert/main.py

```diff
@@ -207,7 +214,22 @@ def _get_source_dir_proposals():
         yield "$WINEPREFIX/" + _STANDARD_PATH_IN_64BIT_WINEPREFIX
     yield "~/.wine/" + _STANDARD_PATH_IN_32BIT_WINEPREFIX
     yield "~/.wine/" + _STANDARD_PATH_IN_64BIT_WINEPREFIX
-    # TODO: wine reg query "HKLM\Software\Microsoft\Microsoft Games\Age of Empires\2.0"
+    try:
+        # get wine registry key of the age installation
+        tmp_reg_file = 'aoe_temp.reg'
+        if subprocess.call(['wine', 'regedit', '/E', tmp_reg_file,
+                        _REGISTRY_KEY]) and os.path.exists(tmp_reg_file):
```

Comment:
as I read tempfile docs, they always construct *and open* a temp file for you, and remove it once you close it. Since we want another process (wine regedit) to write to it, we cannot use tempfile, because python would hold a write fd locking the file